modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -738,9 +740,8 @@
    {database-id                                                 :database
     {source-table-id :source-table, source-query :source-query} :query
     :as                                                         outer-query}]
-  (let [dataset-id         (-> (qp.store/database) :details :dataset-id)
-        {table-name :name} (some-> source-table-id qp.store/table)]
-    (assert (seq dataset-id))
+  (let [{table-name :name, dataset-id :schema} (some-> source-table-id qp.store/table)]
+    #_(assert (seq dataset-id))
```

Comment:
this assertion sounds weird to me. `(seq id)` is strange. Do we want this assertion that dataset-id is a non-empty string? Or was this just testing?

Reply:
I commented this out, but really it should have just been removed.  It was asserting that the `dataset-id` (a String, here) is non-empty.